src/release/breaking-changes/wide-gamut-framework.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Migration guide for wide gamut Color
+description: Description on Color changes for wide gamut support and migration
+             instructions.
+---
+
+## Summary
+
+The API for the [`Color`][] class in `dart:ui` is changing to support wide gamut
+colors.
+
+## Context
+
+Wide gamut color support was [added to the renderer][] previously. That support
+is now being plumbed up [to the Flutter Framework][] so those colors can be used
+by UIs.
+
+## Description of change
+
+Changes to [`Color`][]:
+
+ 1. Adds a enum field which will specify its [`ColorSpace`][].
+ 1. Adds API to use normalized floating-point color components.
```

Comment:
> to use normalized floating-point color components

Perhaps:

> to use normalized floating-point color components (0.0 to 1.0)

Reply:
I don't want to say 0 to 1 because it's technically not true.  The extended srgb colorspace is normalized across sRGB, not its limits.